src/pages/ESNMembersTab/EsnersSheet.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ *
+ * the below is for running the example in the context of this repo.
+ *
+ * for your app, use:
+ *
```

Comment:
Please don't leave comments like this in code

Btw. Why didn't you` import { withGoogleSheets } from 'react-db-google-sheets';` ?

- Thus, I believe you could avoid adding few js files -> As I understand those files (from /dist/) are the content of package `react-db-google-sheets` , which should be imported, not just pasted to repo.
- Apart from that, if you use npm version (so with proper version), this package will be downloaded from npm and stored in node_modules which is one of the fundamentals of all apps based on nodejs.


https://www.npmjs.com/package/react-db-google-sheets

Reply:
Wiem, ale npm w ogóle nie importował mi tej paczki, więc dodałem pliki do dista, nawet Ci wysyłałem ten problem. no a komentarze spoko, moge usunac, bo nie zauważyłem ich